infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/collections/impl/SszBitvectorImpl.java

```diff
@@ -74,8 +74,8 @@ public SszBitvector rightShift(int n) {
   }
 
   @Override
-  public List<Integer> getAllSetBits() {
-    return value.streamAllSetBits().boxed().collect(Collectors.toList());
+  public IntList getAllSetBits() {
+    return IntArrayList.toList(value.streamAllSetBits().boxed().mapToInt(Integer::intValue));
```

Comment:
This is boxing and then unboxing the ints unnecessarily.  Should just be:
```suggestion
    return IntArrayList.toList(value.streamAllSetBits());
```

Reply:
Nice spot. That was a bit redundant. Will fix.